content/docs/lists-and-keys.md

```diff
@@ -70,9 +70,9 @@ ReactDOM.render(
 );
 ```
 
-When you run this code, you'll be given a warning that a key should be provided for list items. A "key" is a special string attribute you need to include when creating lists of elements. We'll discuss why it's important in the next section.
+Cuando ejecutes este codigo, Seras advertido que una key debe ser proporcionada para listar items. Una "key" es un atributo especial string que debes incluir al crear listas de elementos. Vamos a discutir porque esto es importante en la proxima seccion.
```

Comment:
An uppercase `S` made its way into `serás`.
This is how I would write the following sentence:
 > serás advertido que se debería proporcioar una key para elementos de lista.

Note how the original doc says that we **should**, which is not the same as saying that we **must** use keys, hence the `debería`, and that it also says `list items` rather that `listing items`, hence using `elementos de lista` instead of `para listar elementos`.

What do you think?

Reply:
Following up on my last comment, 
> ... es un atributo especial string que ...
is perhaps too much of a literal translation when specifying the type of "key".
If write this instead:
> ... es un atributo especial **de tipo string** que ...

And finally, this is just a personal opinion, but I think `Discutiremos` is better than `Vamos a discutir` on this specific scenario.

Reply:
Thanks for your suggestions. I agree with you